src/App.js

```diff
@@ -1,27 +1,8 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
```

Comment:
- [Optional] Hi @emyrue, please let's make this component class-based to fulfill the requirements of this project.